litestar/cli/_utils.py

```diff
@@ -10,16 +10,32 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Generator, Iterable, Sequence, TypeVar, cast
 
-from click import ClickException, Command, Context, Group, pass_context, style
-from rich.console import Console
+from rich import get_console
 from rich.table import Table
 from typing_extensions import Concatenate, ParamSpec
 
 from litestar import Litestar, __version__
 from litestar.middleware import DefineMiddleware
 from litestar.utils import get_name
 
+RICH_CLICK_INSTALLED = False
+try:
+    import rich_click  # noqa: F401
+
+    rich_click_installed = True
```

Comment:
This is my first look at code in awhile, but is this meant to be the only lowercase instance of `rich_click_installed`?

Reply:
great catch